bin/pos-cli-modules-version.js

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+
+const program = require('commander');
+
+const semver = require('semver');
+
+const logger = require('../lib/logger'),
+  report = require('../lib/logger/report'),
+  files = require('../lib/files'),
+  settings = require('../lib/settings'),
+  dir = require('../lib/directories');
+
+
+function crateNewVersion(moduleName, version, options) {
+  const modulePath = `${dir.MODULES}/${moduleName}`
+  const infoPath = `${modulePath}/template-values.json`;
+  let moduleInfo = settings.loadSettingsFileForModule(moduleName);
+
+  // Read the version from the package file.
+  if (options.package) {
+    let packageJSONPath = `${modulePath}/package.json`;
+    if (typeof options.package === 'string') {
+      packageJSONPath = `${modulePath}/${options.package}`;
+    }
+    version = files.readJSON(packageJSONPath, { throwDoesNotExistError: true }).version;
+  }
+
+  // Validate versions.
+  if (!semver.valid(moduleInfo.version)) {
```

Comment:
I think it should create `template-values.json` if the file does not exists. I've tried to test it and after initializing module with `pos-cli modules init` I had to create `template-values.json` manually.

Reply:
@piotrze thanks for checking this. We haven't modified it in the module template repository that's why you had to create it manually. But you I right, I will create a PR on that repository.